src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -489,7 +487,59 @@ select game
                 tournamentSсheduleInfo);
         }
 
-        private void ValidateGameInRoundOnCreate(
+        private static void IsFreeDayGameValidation(bool GameValidationIsFreeDayGame, TournamentScheduleDto tournamentScheduleInfo, GameResultDto game)
+        {
+            if (GameValidationIsFreeDayGame)
+            {
+                if (tournamentScheduleInfo.Scheme != TournamentSchemeEnum.PlayOff)
+                {
+                    throw new ArgumentException(
+                    Resources
+                    .SameFreeDayGameInRound);
+                }
+                else
+                {
+                    throw new ArgumentException(
+                        string.Format(Resources.SameTeamInRound, game.HomeTeamName));
+                }
+            }
+            else
+            {
+                throw new ArgumentException(
+                   string.Format(
+                   Resources.SameGameInRound,
+                   game.HomeTeamName,
+                   game.AwayTeamName,
+                   game.Round.ToString()));
+            }
+        }
```

Comment:
Add empty line after closing curly bracket.

Reply:
ok

Reply:
It have already fixed.